codespell_lib/data/dictionary.txt

```diff
@@ -21595,6 +21599,7 @@ includied->included
 includig->including
 includign->including
 includng->including
+includs->includes
 inclue->include
 inclued->included
```

Comment:
Perhaps change to:
```
inclued->included, include,
```

Reply:
Done. I also added `incluedd->included` and `inclueds->includes`